### PR TITLE
New wireframe Vulnerabilities CVE Table Toolbar design

### PR DESCRIPTION
**Pull request by janelznic**
@karelhala Please check.

### CODE REVIEW

File: src/PresentationalComponents/SimpleTableFilter/simple-table-filter.scss [hunk 1]
```diff
@@ -9,8 +9,9 @@
         @include rem('margin-right', -23px);
     }
     .ins-c-search-icon {
+        color: #72767b;
```
Comment (karelhala) [inline src/PresentationalComponents/SimpleTableFilter/simple-table-filter.scss]: Minor: Please use css vars so we can easilly change themes
```css
color: var(--pf-global--secondary-color--100)
```

File: src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js [hunk 2]
```diff
@@ -71,19 +71,24 @@ class VulnerabilitiesCveTableToolbar extends Component {
             <TableToolbar>
                 <Grid className="cvetable-toolbar" gutter={ 'md' }>
                     <GridItem span={ 3 }>
-                        <SimpleTableFilter onFilterChange={ value => this.changeFilterValue(value) } buttonTitle={ null } />
+                        <SimpleTableFilter
+                            onFilterChange={ value => this.changeFilterValue(value) }
```
Comment (karelhala) [inline src/SmartComponents/Inventory/applications/vulnerabilities/VulnerabilitiesCveTableToolbar.js]: Nitpick: No need to create new function
```JSX
onFilterChange={ this.changeFilterValue }
```

**Comment by janelznic** [comment]
Snapshot added and commit rebased. Please check and submit.

**Comment by karelhala** [comment]
@jiridostal OK to merge?

**Comment by jiridostal** [comment]
Yes, LGTM.

**Pull request merged**